packages/castle/src/each-sized-batch.ts

```diff
@@ -0,0 +1,52 @@
+import { CastleConsumerConfig, CastleEachBatchPayload } from './types';
+import chunk = require('lodash.chunk');
+
+export default <T extends any>(consumerConf: CastleConsumerConfig<T>): CastleConsumerConfig<T> => {
```

Comment:
Defaults in TS are largely shunned because they are harder to refactor.
https://stackoverflow.com/questions/33305954/typescript-export-vs-default-export

Reply:
Fair enough.